test/Dialect/Tpp/tpp-memory-effects.mlir

```diff
@@ -0,0 +1,8 @@
+// RUN: tpp-opt %s -cse | FileCheck %s
+
+// CHECK-LABEL: pure_at_tensor
+func.func @pure_at_tensor(%arg0: tensor<2x2xf32>) {
```

Comment:
A bit light on tests... Would be nice to have the ones that do work, unary, binary and gemm, too.